### ISSUE TITLE
Feature Request: Place empty vehicle when pressing ALT during placement

### ISSUE THREAD

**Issue by diwako** [zen-mod/ZEN#32]
Since it is possible to hide empty vehicles on the empty side, would it be possible to use 3den's mechanic to place empty vehicles?
That is by holding "alt" during the placement of the vehicle. It would the just spawn the vehicle without crew.
I am aware  that there is the crew toggle, tho it would streamline interactions you'd know when using the 3den editor and now zeus,

**Comment by CreepPork** [comment]
ALT on its own isn't the best in Zeus. If you ALT-TAB then the key will be stuck and you'll have to press it a couple of times to make it reset.

**Comment by diwako** [comment]
Bindable key would be best, if ALT is bad for some, they could use CTRL or any other button they want. In general I do not see an issue with using ALT for this feature, alt tabbing was always an issue in zeus and eden, but you do not really alt tab when placing down units?

**Comment by CreepPork** [comment]
Yeah, I agree about the bindable key, that's fine. But when pressing the ALT (or whatever) and then tab, it will be stuck. So if you don't also show this case to the user (by not changing the checkbox in the tree view) then they'll be dead confused. This issue crops up in Achilles if you use the key as ALT. There you can see the issue.
